### DISCUSSION TITLE
Order of publications within an author search request

### DISCUSSION THREAD

**Discussion by NoahHodgson** [scholarly-python-package/scholarly#440] [Q&A]
Hi there, I was wondering what order the publications are when you get the author JSON back? I originally thought they were ordered by date, but they are not and also the only part of a date associated with them is a year. Just curious about the ordering. Thanks,

-Noah

**Comment by arunkannawadi** [comment]
They'd be in whatever order Google Scholar returns them and that's sorted by "relevance" by default. If you'd like them sorted by date, then include `sort_by = "date"` in your call to `search_pubs`.
